applications/StructuralMechanicsApplication/custom_elements/truss_element_3D2N.cpp

```diff
@@ -656,6 +656,36 @@ void TrussElement3D2N::WriteTransformationCoordinates(
   KRATOS_CATCH("");
 }
 
+void TrussElement3D2N::AddExplicitContribution(
+    const VectorType& rRHSVector,
+    const Variable<VectorType>& rRHSVariable,
+    Variable<double >& rDestinationVariable,
+    const ProcessInfo& rCurrentProcessInfo
+    )
+{
+    KRATOS_TRY;
+
+    auto& r_geom = GetGeometry();
+
+    if (rDestinationVariable == NODAL_MASS) {
+        Matrix element_mass_matrix = ZeroMatrix(msLocalSize, msLocalSize);
+        ProcessInfo temp_info; // Dummy

```

Comment:
do not create a processinfo!!

Reply:
@RiccardoRossi see the current implementation. There is not an alternative implementation. This is due to the element.h (and condition.h) implementation. If we fix that I will agree to remove the ProcessInfo creation, but this is a different discussion (and a discussion affecting to the @KratosMultiphysics/technical-committee )

Reply:
the issue is fixed in #2993 
Anyway you cannot modify the ProcessInfo in the Element (race-condition in OMP)

=> use the `Properties` to set `USE_LUMPED_MASS_MATRIX` (same as for all the other variables ...)